### PR TITLE
fix(pkg): Add project metadata long description (readme)

### LINKED ISSUE #240: Missing Project Description on PyPI
### Summary

I was checking out scicookie on PyPI and found out that there is currently no project description available.

### Documentation Page Link

https://pypi.org/project/scicookie/

### Code of Conduct

- [X] I agree to follow the Code of Conduct

### PR DESCRIPTION
**Pull request by Satarupa22-SD**
Resolves #240 
@xmnlab

### CODE REVIEW

File: docs/description.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+![CI](https://img.shields.io/github/actions/workflow/status/osl-incubator/scicookie/main.yaml?logo=github&label=CI)
+[![Python Versions](https://img.shields.io/pypi/pyversions/scicookie)](https://pypi.org/project/scicookie/)
+[![Package Version](https://img.shields.io/pypi/v/scicookie?color=blue)](https://pypi.org/project/scicookie/)
+![License](https://img.shields.io/pypi/l/scicookie?color=blue)
+![Discord](https://img.shields.io/discord/796786891798085652?logo=discord&color=blue)
+
+![logo_scicookie.png](images%2Flogo_scicookie.png)
+
+**SciCookie** is a Cookiecutter template designed to simplify scientific Python project creation. It provides a boilerplate structure with recommended tools, workflows, and industry best practices, saving developers time and effort. Built upon the PyOpenSci recommendations, it offers a foundation that adheres to scientific Python standards while remaining customizable to specific project needs.
```
Comment (xmnlab) [inline docs/description.md:9]:
```suggestion
**SciCookie** is a project template designed to simplify scientific Python project creation. It provides a boilerplate structure with recommended tools, workflows, and industry best practices, saving developers time and effort. Built upon the PyOpenSci recommendations, it offers a foundation that adheres to scientific Python standards while remaining customizable to specific project needs.
```

**Comment by xmnlab** [comment]
thanks @Satarupa22-SD for working on that, appreciate that.

I just added a small comment here.

other than that, please run `pre-commit install` in your scicookie environment and run `pre-commit run --all-files` to fix the issue that is raising on CI.

for more information, please take a look here: https://osl-incubator.github.io/scicookie/contributing/#setting-up-for-local-development.

note: maybe we would need to discuss a bit more the logo later, check if that works fine with favicon etc .. but for now it looks great. thanks!

**Comment by Satarupa22-SD** [comment]
@xmnlab I have updated the description.md file.

**Comment by xmnlab** [comment]
Later we would need to discuss about README.md, docs/index.md and the docs/description.md .. we need to simplify that .. otherwise we will have eventually outdated info for forgetting to update one of these files.

**Comment by Satarupa22-SD** [comment]
@xmnlab sure :)

**Pull request merged**